src/_product-updates/2025-07-15-documentation-fixes-and-improvements.md

```diff
@@ -0,0 +1,31 @@
+---
+title: Documentation fixes and improvements
+tags: [Improvement]
+---
+Even if our product grows with new features, taking the time to maintain, enhance, and polish existing features is crucial to ensure a solid documentation experience. That's why we always give ourselves time to tackle small enhancements and fixes alongside the development of new features. Here's a list of what's been worked on.
+
+Improvements:
+- You can now add more than one additional link to the navigation bar by using [our new custom extension](https://docs.bump.sh/help/publish-documentation/external-links/) called `x-externalLinks` instead of the `externalDocs` property,
+- When the link to an operation or group no longer exists, it now returns a 404 page instead of redirecting to the top of the documentation, providing a clearer feedback,
```

Comment:
I don't remember if this improvement has already been announced, but i could be nice to mention that current operation is maintained by switching between several branches (if operation exists only).